pkg/account/handler.go

```diff
@@ -103,6 +113,60 @@ func (h *AccountHandler) CreateAccount(c echo.Context) error {
 	}))
 }
 
+// @Summary set doctor info
+// @Description
+// @Tags Account
+// @Produce json
+// @Param Department path string true "doctor department"
+// @Success 200 {string} api.ReturnedData{data=nil}
+// @Failure 400 {string} api.ReturnedData{data=nil}
+// @Router /account/create [POST]
```

Comment:
```suggestion
// @Router /account/setdoctor [POST]
```

besides, set doctor is an awful name. What does it set?